spec/osl_mysql_dev.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe 'osl_mysql_dev' do
+  # Test the initalization of the sql service with a database
+  # then add another database as well.
+
+  recipe do
+    osl_mysql_dev 'db-one' do
+      username 'foo'
+      password 'bar'
+    end
+
+    osl_mysql_dev 'db-two' do
+      username 'foo'
+      password 'bar'
+    end
+  end
+
+  context 'centos' do
+    platform 'centos'
+    cached(:subject) { chef_run }
+    step_into :mysql_test_db
+
+    # mysql_temp create action.
+
+    it {
+      is_expected.to install_mariadb_server_install('default').with(
+      encoding: 'utf8mb4',
+      collation: 'utf8mb4_unicode_ci'
+    )
+    }
```

Comment:
The use of `it { foo }` is only for one-liners. When doing multi-line please do it with `it do` with an `end` on the last line. Also, please format it like the following for the other lines:
```suggestion
    it do
      is_expected.to install_mariadb_server_install('default').with(
        encoding: 'utf8mb4',
        collation: 'utf8mb4_unicode_ci'
      )
    end
```